projects/index.html

```diff
@@ -109,7 +108,122 @@ <h2 class="to-animate hero-animate-2" style="margin-bottom:10px; color:black"><s
 		<div id="fh5co-main">
 			<div class="container">
 
+				<!-- The following contents are for md and lg screen -->
+				<div class="visible-md visible-lg">
+				  <div class="row" id="fh5co-works">
+					<div class="col-md-8 col-md-offset-2 text-center fh5co-section-heading work-box">
+						<h2 class="fh5co-lead">About this course</h2>
+						<p class="fh5co-sub">As data becomes increasingly important in news reporting, this course, which mainly focuses on data visualisation while covering basic data analytics and data journalism workflow, helps students learn the fundamental principles, tools and working process of data news visualization.</p>
+                        <p class="fh5co-sub">At the end of this course, students are expected to build one web-based data visualisation news story and distribute the project via their own websites. Here're the the great stories that the students did in 2017.</p>
+						<div class="fh5co-spacer fh5co-spacer-sm"></div>
+					</div>
+					<div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box">
```

Comment:
`col-sm-8 col-xs-8 col-xxs-12` should have no effect here since this block is only visible in on md/lg screens. Please verify and update the code accordingly.